src/lib/auth.ts

```diff
@@ -255,10 +255,27 @@ export const authenticate = (appMode: AppMode, appConfig: AppConfigModelType, ur
       if (!fakeClass || !fakeUser) {
         return reject("Missing fakeClass or fakeUser parameter for demo!");
       }
-      const [userType, userId] = fakeUser.split(":");
+      const userType = fakeUser.split(":")[0];
+      let userId = fakeUser.split(":")[1];
```

Comment:
Rather than breaking this into two lines, which seems inelegant, I'd recommend this:
```suggestion
      let [userType, userId] = fakeUser.split(":");
```
and then change the `prefer-const` definition in `.eslintrc.js` to:
```
      "prefer-const": ["error", {"destructuring": "all"}],
```
This makes it okay for only some destructured variables to be reassigned. I've used this setting in other projects and prefer it.